src/window.rs

```diff
@@ -139,6 +140,11 @@ impl Window {
             .or_else(|_| create_gl_window(window_builder, &event_loop, true))?;
         window.show();
 
+        // Maximize window after mapping in X11
+        if cfg!(not(any(target_os = "macos", windows))) && event_loop.is_x11() {
```

Comment:
We should use `#[cfg(...)]` here instead of `cfg!`. It's a little uglier since it requires an extra scope, but that makes sure it's only checked at compile-time.

Reply:
Oh, OK, thanks - yeah, on MacOS and Windows it was failing because of that.